lib/objects.rb

```diff
@@ -186,12 +186,6 @@ def get_all_file_info(object)
   def get_file_info(object, file)
     # TODO: Split me up further once file parsing is finalized
     file_hash = {}
-    file_info = {}
-    begin
-      file_info = Hash.from_xml(file['structure'])['Item']
-    rescue
-      object_error_and_exit(object, "failed to parse item xml")
-    end
 
     # Setup the defaults
     file_hash[:workflow_name] = 'avalon'
```

Comment:
We should probably change this default to 'mdpi' or something like that or make it configurable via a yaml file.